Classes/Domain/Import/UrlProvider/ContainsPlaceUrlProvider.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * Copyright (C) 2021 Daniel Siepmann <coding@daniel-siepmann.de>
```

Comment:
This class seems to be completely new. So `2022`?

```suggestion
 * Copyright (C) 2022 Daniel Siepmann <coding@daniel-siepmann.de>
```